odl/tomo/geometry/geometry.py

```diff
@@ -45,18 +45,30 @@ class Geometry(object):
     <https://odlgroup.github.io/odl/guide/geometry_guide.html>`_.
     """
 
-    def __init__(self, ndim, motion_part, detector, translation=None):
+    def __init__(self, ndim, motion_part, detector, translation=None,
+                 **kwargs):
         """Initialize a new instance.
 
         Parameters
         ----------
         ndim : positive int
             Number of dimensions of this geometry, i.e. dimensionality
-            of the physical space in which this geometry is embedded
+            of the physical space in which this geometry is embedded.
         motion_part : `RectPartition`
-           Partition for the set of "motion" parameters
+            Partition for the set of "motion" parameters.
         detector : `Detector`
-           The detector of this geometry
+            The detector of this geometry.
+        translation : `array-like`, optional
+            Global translation of the geometry. This is added last in any
+            method that computes an absolute vector, e.g., `det_refpoint`.
+            Default: zero vector of length ``ndim``
+
+        Other Parameters
+        ----------------
+        check_bounds : bool, optional
+            If ``True``, methods perform sanity checks on provided input
```

Comment:
Someone ambiguous what parameters are refered to here, I'd read it as the parameters to this method.